packages/react/src/components/Embed/Embed.tsx

```diff
@@ -134,6 +142,7 @@ class Embed extends AutoControlledComponent<WithAsProp<EmbedProps>, EmbedState>
         {...unhandledProps}
         {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.root, unhandledProps)}
       >
+        {iframe && active && !iframeLoaded && placeholderImage}
```

Comment:
What does this condition? Is there way to simplify rendering there?
There is too many conditions to be honest.

Reply:
I have extracted it into a function, please check again